### DISCUSSION TITLE
Segmentation Fault in Installation

### DISCUSSION THREAD

**Discussion by 18rway** [iraf-community/iraf#146] [Q&A]
Hello,
I am attempting to install IRAF on my Mac with little experience with a computer thus far. Following the instructions on the installation page, I executed make sysgen 2>&1 | tee build.log, but now I am hit with an error that says:
file too small (length=8) file '/Users/Ulrich/iraf-2.16.1-2018.11.01/lib/libsys.a' for architecture x86_64
clang: error: unable to execute command: Segmentation fault: 11
clang: error: linker command failed due to signal (use -v to see invocation) 
Does anyone know what this means/could help with figuring this out? Anything would be appreicated.
Best,
Raphael

**Comment by olebole** [comment]
Can you attach the complete log (`build.log`)?

**Comment by olebole** [comment]
A segmentation fault in the compiler or the linker is much more probably a problem with the compiler/linker, or the setup that with the source. I think you should check your setup that it is correct and working well. One can't say more without looking in the build log.
Therefore, I am closing this. If it is a problem of the sources, we can re-open.
